recipes-graphics/libepoxy/libepoxy_1.4.0.bb

```diff
@@ -1,4 +1,8 @@
 SUMMARY = "OpenGL function pointer management library"
+DESCRIPTION = "Epoxy is a library for handling OpenGL function pointer management \
```

Comment:
Could you please indent multiline with 4 spaces?

Reply:
done